File: 1512-number-of-good-pairs/1512-number-of-good-pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int temp[]=new int[101]; //nums.length<=100
            int res=0;
            for(int a:nums){
                    res+=temp[a];
                    temp[a]++;
            }
            return res;
            
    }
}

File: 1512-number-of-good-pairs/1512-number-of-good-pairs.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        vector<int>v(101,0);
        int c=0;
        for(int i:nums){
            c+=v[i];
            v[i]++;
        }
        return c;
    }
};